atuin/src/command/client/sync.rs

```diff
@@ -62,12 +71,26 @@ impl Cmd {
     }
 }
 
-async fn run(settings: &Settings, force: bool, db: &mut impl Database) -> Result<()> {
+async fn run(
+    settings: &Settings,
+    force: bool,
+    db: &mut impl Database,
+    store: &mut (impl Store + Send + Sync),
+) -> Result<()> {
+    let (diff, remote_index) = sync::diff(settings, store).await?;
+    let operations = sync::operations(diff, store).await?;
+    let (uploaded, downloaded) =
+        sync::sync_remote(operations, &remote_index, store, settings).await?;
+
+    println!("{uploaded}/{downloaded} up/down to record store");
```

Comment:
records won't sync automatically here

Reply:
intentional atm

Reply:
I don't want to piggy back it on top of the old record sync triggers, would like to do something better